src/main/java/com/google/api/gax/protobuf/PathTemplate.java

```diff
@@ -209,27 +209,27 @@ String separator() {
   }
 
   /**
-   * Creates a path template from a string. The string must satisfy the syntax
-   * of path templates of the API platform; see HttpRule's proto source.
+   * Creates a path template from a string. The string must satisfy the syntax of path templates of
+   * the API platform; see HttpRule's proto source.
    *
    * @throws ValidationException if there are errors while parsing the template.
    */
-  public static PathTemplate create(String template) {
+  public static PathTemplate create(String template) throws ValidationException {
```

Comment:
Yes, I got tricked :-(
Somehow I had made Eclipse import javax.xml.bind.ValidationException for a brief moment, and it convinced me to add the throws declarations...